### PR TITLE
Move systemtags to the new bootstrap mechanism

### PR DESCRIPTION
**Pull request by ChristophWurst**

### CODE REVIEW

File: apps/systemtags/lib/AppInfo/Application.php [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+		$mapperListener = function (MapperEvent $event) use ($context) {
+			/** @var \OCA\SystemTags\Activity\Listener $listener */
+			$listener = $context->getServerContainer()->query(Listener::class);
+			$listener->mapperEvent($event);
+		};
+		$eventDispatcher->addListener(MapperEvent::EVENT_ASSIGN, $mapperListener);
+		$eventDispatcher->addListener(MapperEvent::EVENT_UNASSIGN, $mapperListener);
+
+		\OCA\Files\App::getNavigationManager()->add(function () {
```
Comment (rullzer) [inline apps/systemtags/lib/AppInfo/Application.php:77]: Note to self we should move this as well probably.

File: apps/systemtags/lib/AppInfo/Application.php [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+	}
+
+	public function register(IRegistrationContext $context): void {
+	}
+
+	public function boot(IBootContext $context): void {
+		/*
+		 * @todo move the OCP events and then move the registration to `register`
+		 */
+		$eventDispatcher = $context->getServerContainer()->getEventDispatcher();
```
Comment (rullzer) [inline apps/systemtags/lib/AppInfo/Application.php:50]: And we should get rid of those old events I guess

**Review by rullzer — APPROVED** [review]
Did not test but looks good!

**Review by MorrisJobke — APPROVED** [review]
Tested and works 👍

**Pull request merged**